subprojects/ide-play/src/main/java/org/gradle/play/plugins/ide/PlayIdePlugin.java

```diff
@@ -28,7 +27,6 @@
  *
  * <p>NOTE: This currently supports configuring the 'idea' plugin only.</p>
  */
-@Incubating
```

Comment:
I probably wouldn't promote the play stuff yet, as it depends on the software model.

Reply:
Not promoting play any more.